mux/bindings/generate.sh

```
@@ -0,0 +1,99 @@
+    echo "- Regeneration must overwrite stale generated files inside the temporary output directory."
+    echo "- Follow the style sheet exactly."
+    echo "- Implement protocol-v5 commands marked implemented, plus consumer-side implemented move commands."
+    echo "- Preserve command names, params, result shapes, event names, and error behavior from the spec."
+    echo "- Do not implement proposed commands unless the style sheet explicitly asks for version-gated stubs."
+    echo
+    echo "## Language style sheet"
+    cat "$style"
+    echo
+    for spec in "${SPEC_FILES[@]}"; do
+      echo
+      echo "## Spec: ${spec#$REPO_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)